README.md

```diff
@@ -29,6 +29,14 @@ joining our [public Slack channel.](https://adhocteam-public.herokuapp.com)
 We think a sign of a good developer is one who asks questions sooner, rather
 than later. We are happy to help guide you to the right solution.
 
+On Overengineering
```

Comment:
But that's just bad programming, right? I'm explicitly just trying to cover the issue of overengineering here because I think going above and beyond is both a reasonable approach and something we take points off for.